### PR TITLE
Remove cron job for building 1.3.18

### PR DESCRIPTION
**Pull request by zelinh**
### Description
Remove cron job for building 1.3.18

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/4763

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by gaiksaya — APPROVED** [review]

**Comment by zelinh** [comment]
I will merge this PR once I confirmed all pending commits are merged.

**Pull request merged**
